samigo/samigo-hibernate/src/java/org/sakaiproject/tool/assessment/data/dao/assessment/AssessmentBaseData.java

```diff
@@ -356,7 +357,7 @@ public String getAssessmentMetaDataByLabel(String label) {
   }
 
   public void addAssessmentMetaData(String label, String entry) {
-    if (this.assessmentMetaDataMap.get(label)!=null){
+      if (this.assessmentMetaDataMap.containsKey(label)) {
```

Comment:
Bad spacing

Reply:
Nooooooooooooo

![image](https://github.com/sakaiproject/sakai/assets/8653754/7e5b2a82-e475-4c76-903e-43d9fb5fb7fc)

Reply:
Not sure I understand the screenshot, surely that "if" and the following lines shouldn't have the same indentation?

Reply:
You're right, reverted